src/lmms_engine/models/qwen3_moe/qwen3_moe_ops.py

```
@@ -0,0 +1,312 @@
+            past_seen_tokens + seqlen,
+            device=inputs_embeds.device,
+        )
+
+    if position_ids is None:
+        position_ids = cache_position.unsqueeze(0)
+    position_ids = position_ids.repeat_interleave(bs, dim=0)
+
+    position_ids = index_first_axis(
+        rearrange(position_ids.unsqueeze(-1), "b s ... -> (b s) ..."), indices
+    ).transpose(0, 1)
+    original_position_ids = position_ids
```

Ruff (0.12.2)

Local variable `original_position_ids` is assigned to but never used

Remove assignment to unused variable `original_position_ids`

(F841)